.github/workflows/CI.yml

```diff
@@ -1061,12 +1060,16 @@ jobs:
           aws-region: us-west-2
 
       - name: restore cached packages
+        id: cache_packages
         uses: actions/cache@v2
         with:
           path: "${{ env.PACKAGING_ROOT }}"
           key: "cached_packages_${{ github.run_id }}"
 
       - name: clean up SSM test package
+        if: steps.cache_packages.outputs.cache-hit == 'true'
         run: |
-          ssm_pkg_version=`cat $PACKAGING_ROOT/VERSION`
-          aws ssm delete-document --name "testAWSDistroOTel-Collector" --version-name ${ssm_pkg_version}
+          ssm_pkg_version=`cat build/packages/VERSION`
+          aws ssm describe-document --name ${SSM_PACKAGE_NAME} --version-name ${ssm_pkg_version} >/dev/null 2>&1 && \
+            aws ssm delete-document --name ${SSM_PACKAGE_NAME} --version-name ${ssm_pkg_version}
```

Comment:
so this is probably my lack of yml knowledge but why do we have to write the command twice?

Reply:
We only need to delete (second command) if the ```ssm package name``` exist (first command).

Reply:
oh right, sorry i misread them both as describe